xbmc/platform/Platform.h

```diff
@@ -23,17 +23,33 @@ class CPlatform
   static CPlatform *CreateInstance();
 
   /**\brief C'tor */
-  CPlatform();
+  CPlatform() = default;
 
   /**\brief D'tor */
-  virtual ~CPlatform();
+  virtual ~CPlatform() = default;
 
   /**\brief Called at an early stage of application startup
    *
    * This method can be used to do platform specific environment preparation
    * or initialisation (like setting environment variables for example)
    */
-  virtual bool Init();
+  virtual bool Init() { return true; };
+
+  /**\brief Called at a middle stage of application startup
```

Comment:
Can you note that logging will be available here (it is available in stage one but may not actually log anything because `CLog::Initialize` isn't called till after stageone. Platforms that use special loggers (android, osx) may show the logging from initstageone but other platforms that only use the log file logging won't.